riptide-stream/src/main/java/org/zalando/riptide/stream/StreamSpliterator.java

```diff
@@ -86,4 +86,13 @@ public long estimateSize() {
     public int characteristics() {
         return ORDERED | IMMUTABLE;
     }
+
+    @Override
+    public void close() throws UncheckedIOException {
```

Comment:
> JsonParser will close the stream correctly per default

Yes, but if somebody changes the configuration we are not behaving correctly and since we are not exposing a way to close the stream the client is unable to workaround that issue.